examples/android/libKtAndroid/src/main/java/examples/android/lib/TestKtValue.kt

```diff
@@ -5,11 +5,17 @@ import com.google.auto.value.AutoValue
 @AutoValue
 abstract class TestKtValue {
   abstract fun name(): String
-  fun builder(): Builder = AutoValue_TestKtValue.Builder()
+  internal fun nameInternal() = "${name()}_internal"
 
   @AutoValue.Builder
   abstract class Builder {
     abstract fun setName(name: String): Builder
     abstract fun build(): TestKtValue
   }
+  companion object {
```

Comment:
Not strictly necessary, but builder() wasn't used, and having a very incorrect use of autovalue bugged me.